src/index.html

```diff
@@ -1,5 +1,5 @@
-<!doctype html>
 <html lang="en">
+<!doctype html>
```

Comment:
The `<!doctype html>` declaration should be the first line in the HTML document, before the `<html>` tag. This ensures that the browser correctly interprets the document as HTML5.